wallet/lib/pages/purchase_item/widgets/pay_now_dialog.dart

```diff
@@ -83,39 +78,31 @@ class PayNowWidget extends StatefulWidget {
 class _PayNowWidgetState extends State<PayNowWidget> {
   @override
   Widget build(BuildContext context) {
-    final fee = double.parse(widget.nft.price) * 0.1;
-    final price = double.parse(widget.nft.price) - fee;
-
     return Container(
       color: Colors.black.withOpacity(0.7),
-      height: isTablet ? 290.h : 340.h,
+      height: 390.h,
+      width: isTablet ? 200.w : 270.w,
       margin: isTablet ? EdgeInsets.symmetric(horizontal: 30.w) : EdgeInsets.zero,
       child: Stack(
         children: [
           Positioned(
             right: 0,
-            bottom: 0,
-            child: SizedBox(
-              height: 60,
-              width: 80,
-              child: ClipPath(
-                clipper: RightTriangleClipper(orientation: enums.Orientation.Orientation_NW),
-                child: ColoredBox(
-                  color: AppColors.kDarkRed,
-                ),
-              ),
-            ),
-          ),
-          Positioned(
-            left: 0,
             top: 0,
             child: SizedBox(
               height: 60,
               width: 80,
               child: ClipPath(
-                clipper: RightTriangleClipper(orientation: enums.Orientation.Orientation_SE),
-                child: ColoredBox(
+                clipper: RightTriangleClipper(orientation: enums.Orientation.Orientation_SW),
+                child: Container(
                   color: AppColors.kDarkRed,
+                  alignment: Alignment.topRight,
+                  child: GestureDetector(
+                    onTap: () => Navigator.pop(context),
+                    child: Container(
+                        padding: EdgeInsets.only(right: isTablet ? 2.w : 5.w, top: 5.h),
+                        child: Image(image: const AssetImage("assets/images/icons/close_white.png"), height: isTablet ? 15.h : 18.h, width: isTablet ? 15.w : 18.h,),
```

Comment:
We dont direclty add the assets paths in the code. It should be in a constant file and then it should be use here. Kindly check constant file.

Reply:
Got it